### ISSUE TITLE
Toolkit Quick-Start Guide dosn't work for me on Synology916+ with DSM 7

### ISSUE THREAD

**Issue by east-mist** [overleaf/toolkit#79]
## Steps to Reproduce
I am new to docker and Linux. 
I followed the Quick-Start Guide to the point 'Starting up'. I run $ bin/up and then...

## Expected Behaviour
Docker compose I think

## Observed Behaviour
After the download processes ended:
`Starting mongo ... error
Starting redis ...

Starting redis ... error

ERROR: for redis  Cannot start service redis: Bind mount failed: '/root/overleaf/data/redis' does not exists

ERROR: for mongo  Cannot start service mongo: Bind mount failed: '/root/overleaf/data/mongo' does not exists

ERROR: for redis  Cannot start service redis: Bind mount failed: '/root/overleaf/data/redis' does not exists
ERROR: Encountered errors while bringing up the project.`

## Context
I've run $bin/doctor
`====== Overleaf Doctor ======
- Host Information
    - Linux
    - lsb_release not found !
- Dependencies
    - bash
        - status: present
        - version info: 4.4.23(1)-release
    - docker
        - status: present
        - version info: Docker version 20.10.3, build b455053
    - docker-compose
        - status: present
        - version info: docker-compose version 1.28.5, build 324b023a
    - realpath
        - status: present
        - version info: realpath (GNU coreutils) 8.30
    - perl
        - status: present
        - version info: 5.028003
    - awk
        - status: present
        - version info: GNU Awk 5.0.0, API: 2.0 (GNU MPFR 4.0.1, GNU MP 6.1.2)
    - openssl
        - status: present
        - version info: OpenSSL 1.1.1k  25 Mar 2021
- Docker Daemon
    - status: up
====== Configuration ======
- config/version
    - status: present
    - version: 3.0.1
- config/overleaf.rc
    - status: present
    - values
        - SHARELATEX_DATA_PATH: data/sharelatex
        - SERVER_PRO: false
        - SHARELATEX_LISTEN_IP: 127.0.0.1
        - SHARELATEX_PORT: 80
        - MONGO_ENABLED: true
        - MONGO_DATA_PATH: data/mongo
        - REDIS_ENABLED: true
        - REDIS_DATA_PATH: data/redis
        - NGINX_ENABLED: false
        - NGINX_CONFIG_PATH: config/nginx/nginx.conf
        - TLS_PRIVATE_KEY_PATH: config/nginx/certs/overleaf_key.pem
        - TLS_CERTIFICATE_PATH: config/nginx/certs/overleaf_certificate.pem
        - NGINX_HTTP_LISTEN_IP: 127.0.1.1
        - NGINX_HTTP_PORT: 80
        - NGINX_TLS_LISTEN_IP: 127.0.1.1
        - TLS_PORT: 443
- config/variables.env
    - status: present
====== Warnings ======
- None, all good
====== End ======`

## Technical Info
I enabled ssh and I'm using PowerShell on Windows 10 to access my Synology. I've done everything as root@MyServer.

## Analysis
I'm sure its either me doing something incorrectly or my Synology isn't supported by quick start guide and I should've use some other guide.

**Comment by apollotsantos** [comment]
Did you solve this?

**Comment by east-mist** [comment]
> 
> 
> Did you solve this?

I asked help in some discord server and I was advised to create the mongo and redis folders in data.
Then I've got the same error with sharelatex folder. I created it.
Then everything went smooth until during instalation through bin/up mongo pernamently made conection to some port, than disconect, and contected to the next port and so on. After over 100 conection I stoped the process and I gave up until weekends, because I don't have any free time
